.github/workflows/test.yml

```diff
@@ -30,46 +33,33 @@ jobs:
       with:
         python-version: ${{ matrix.python_version }}
 
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install 'coverage<5.0.0' pretend pytest
-      shell: bash
-
-    - name: Test coverage
-      run: |
-        python -m coverage run --source packaging/ -m pytest --strict tests
-        python -m coverage report -m --fail-under 100
-      shell: bash
-
-
-  test-pypy:
-    name: Test ${{ matrix.pypy_version }} on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ubuntu-latest, windows-latest, macOS-latest]
-        pypy_version: ['pypy2', 'pypy3']
-
-    steps:
-    - uses: actions/checkout@v1
-
+    # Use newer Python, if we're not Python 3.6+
+    #    why? nox needs Python 3.5+ and @pradyunsg likes f-strings.
     - uses: actions/setup-python@v1
-      name: Install ${{ matrix.pypy_version }}
+      name: Install Python 3.x
       with:
-        python-version: ${{ matrix.pypy_version }}
+        python-version: 3.x
+      if: >
+        (
+          matrix.python_version == '2.7' ||
+          matrix.python_version == 'pypy2' ||
+          matrix.python_version == '3.5'
+        )
+
+    # Workaround https://github.com/theacodes/nox/issues/250
+    - name: Workaround for Windows Python 2.7
+      # This is in PATH, so nox resolves to it - but then subsequent steps fail
+      run: rm C:/ProgramData/Chocolatey/bin/python2.7.exe
+      shell: bash
```

Comment:
pro tip: `shell: python` should be even more cross-platform :)

https://github.com/pypa/pip/pull/6953/files#diff-2deae8ed35e0da386b702aa047e106cbR118-R131

Reply:
And even in this case, PowerShell has an `rm` alias, so I don't know if specifying any shell is needed.

Reply:
Thanks for the tip! That's definitely useful!

I don't think there's any need to change this specific invocation though -- the `bash` runner is supported on all platforms and, well, this works. If someone can check and verify that removing the `shell: bash` line works, we can make that change in a follow up PR. :)